src/templates/Cart/index.tsx

```diff
@@ -7,6 +7,7 @@ import CartList, { CartListProps } from 'components/CartList'
 import Heading from 'components/Heading'
 import Showcase from 'components/Showcase'
 import Base from 'templates/Base'
+import { ErrorWarning } from '@styled-icons/remix-line/ErrorWarning'
```

Comment:
Hey, instead of using `remix-line`, let's stick with things from `material-outlined`?

We have this one:

```suggestion
import { Info } from '@styled-icons/material-outlined/Info'
```